digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleSpeedLimitDao.scala

```diff
@@ -183,11 +183,12 @@ class OracleSpeedLimitDao(val vvhClient: VVHClient, val roadLinkService: RoadLin
       select s.link_id, m.name_fi, s.administrative_class
       from unknown_speed_limit s
       join municipality m on s.municipality_code = m.id
+      where s.unnecessary = 0
       """
 
     val filterAdministrativeClass = administrativeClass match {
-      case Some(ac) if ac == Municipality => s" where s.administrative_class not in ( ${State.value}, ${Private.value})"
-      case Some(ac) if ac == State => s" where s.administrative_class = ${ac.value}"
+      case Some(ac) if ac == Municipality => s" and s.administrative_class not in ( ${State.value}, ${Private.value})"
+      case Some(ac) if ac == State => s" and s.administrative_class = ${ac.value}"
```

Comment:
does this change break `val unknownSpeedLimitQuery` which also expects a `where`?

Reply:
In line 186 I put a where so this change will not break the query. It will simply add more conditions